meta-xt-prod-cockpit-rcar-driver-domain/recipes-kernel/v4l2loopback-mod/v4l2loopback-mod_0.1.bb

```diff
@@ -0,0 +1,22 @@
+SUMMARY = "v4l2loopback-module"
+DESCRIPTION = "Adds loopback v4l2 devices"
+LICENSE = "GPLv2"
+LIC_FILES_CHKSUM = "file://COPYING;md5=b234ee4d69f5fce4486a80fdaf4a4263"
+SRCBRANCH = "main"
+
+inherit module
+
+SRC_URI = "git://github.com/umlaeute/v4l2loopback.git;protocol=https;branch=${SRCBRANCH} \
+	file://0001-makefile-Adapt-makefile-for-yocto-build.patch \
+"
+SRCREV = "1ecf810f0d687b647caa3050ae30cf51b5902afd"
```

Comment:
This corresponds to almost 7 month old commit, is there a reason to not take latest state?